# FRC 2026 AprilTags converted from WPILib layout JSON
# Note: edit tag_family and tag_size/size if your pipeline differs.
apriltag_ros:
  ros__parameters:
    tag_family: tag36h11
    tag_size: 0.1651
    standalone_tags:
      - id: 1
        size: 0.1651
        pose:
          position: {x: 11.863959, y: 7.411491399999999, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 2
        size: 0.1651
        pose:
          position: {x: 11.9013986, y: 4.6247558, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.7071067811865476, w: 0.7071067811865476}
      - id: 3
        size: 0.1651
        pose:
          position: {x: 11.2978438, y: 4.3769534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 4
        size: 0.1651
        pose:
          position: {x: 11.2978438, y: 4.0213534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 5
        size: 0.1651
        pose:
          position: {x: 11.9013986, y: 3.417951, z: 1.12395}
          orientation: {x: -0.0, y: 0.0, z: 0.7071067811865476, w: -0.7071067811865475}
      - id: 6
        size: 0.1651
        pose:
          position: {x: 11.863959, y: 0.6312154, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 7
        size: 0.1651
        pose:
          position: {x: 11.9388636, y: 0.6312154, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 8
        size: 0.1651
        pose:
          position: {x: 12.2569986, y: 3.417951, z: 1.12395}
          orientation: {x: -0.0, y: 0.0, z: 0.7071067811865476, w: -0.7071067811865475}
      - id: 9
        size: 0.1651
        pose:
          position: {x: 12.5051566, y: 3.6657534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 10
        size: 0.1651
        pose:
          position: {x: 12.5051566, y: 4.0213534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 11
        size: 0.1651
        pose:
          position: {x: 12.2569986, y: 4.6247558, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.7071067811865476, w: 0.7071067811865476}
      - id: 12
        size: 0.1651
        pose:
          position: {x: 11.9388636, y: 7.411491399999999, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 13
        size: 0.1651
        pose:
          position: {x: 16.499332, y: 7.391907999999999, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 14
        size: 0.1651
        pose:
          position: {x: 16.499332, y: 6.960107999999999, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 15
        size: 0.1651
        pose:
          position: {x: 16.4989764, y: 4.3124882, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 16
        size: 0.1651
        pose:
          position: {x: 16.4989764, y: 3.8806881999999994, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 17
        size: 0.1651
        pose:
          position: {x: 4.6490636, y: 0.6312154, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 18
        size: 0.1651
        pose:
          position: {x: 4.6115986, y: 3.417951, z: 1.12395}
          orientation: {x: -0.0, y: 0.0, z: 0.7071067811865476, w: -0.7071067811865475}
      - id: 19
        size: 0.1651
        pose:
          position: {x: 5.2151534, y: 3.6657534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 20
        size: 0.1651
        pose:
          position: {x: 5.2151534, y: 4.0213534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 21
        size: 0.1651
        pose:
          position: {x: 4.6115986, y: 4.6247558, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.7071067811865476, w: 0.7071067811865476}
      - id: 22
        size: 0.1651
        pose:
          position: {x: 4.6490636, y: 7.411491399999999, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 23
        size: 0.1651
        pose:
          position: {x: 4.574159, y: 7.411491399999999, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 24
        size: 0.1651
        pose:
          position: {x: 4.2559986, y: 4.6247558, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 0.7071067811865476, w: 0.7071067811865476}
      - id: 25
        size: 0.1651
        pose:
          position: {x: 4.007866, y: 4.3769534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 26
        size: 0.1651
        pose:
          position: {x: 4.007866, y: 4.0213534, z: 1.12395}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 27
        size: 0.1651
        pose:
          position: {x: 4.2559986, y: 3.417951, z: 1.12395}
          orientation: {x: -0.0, y: 0.0, z: 0.7071067811865476, w: -0.7071067811865475}
      - id: 28
        size: 0.1651
        pose:
          position: {x: 4.574159, y: 0.6312154, z: 0.889}
          orientation: {x: 0.0, y: 0.0, z: 1.0, w: 6.123233995736766e-17}
      - id: 29
        size: 0.1651
        pose:
          position: {x: 0.0136906, y: 0.6507734, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 30
        size: 0.1651
        pose:
          position: {x: 0.0136906, y: 1.0825734, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 31
        size: 0.1651
        pose:
          position: {x: 0.0140462, y: 3.7301932, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
      - id: 32
        size: 0.1651
        pose:
          position: {x: 0.0140462, y: 4.1619931999999995, z: 0.55245}
          orientation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}
